frontend/src/utils/maxLengths.ts

```diff
@@ -0,0 +1,11 @@
+export const MaxLengths = {
```

Comment:
```suggestion
export const MaxLength = {
```
Acho que assim fica melhor, tentar apertar f2 para mudar em todos os arquivos